torch/autograd/grad_mode.py

```diff
@@ -59,3 +59,38 @@ def __enter__(self):
     def __exit__(self, *args):
         torch.set_grad_enabled(self.prev)
         return False
+
+
+class grad_mode(object):
+    r"""Context-manager that sets gradient calculation to on or off.
+
+    Enables gradient calculation inside a :class:`~no_grad` context. This has
+    no effect outside of :class:`~no_grad`.
```

Comment:
I don't think this is true. In general it can be used outside of `no_grad`, or even as a replacement to `no_grad`. I don't think this paragraph is necessary